core/state/statedb.go

```diff
@@ -368,6 +368,24 @@ func (s *StateDB) GetCodeHash(addr common.Address) common.Hash {
 	return common.Hash{}
 }
 
+// ResolveCode retrieves the code at addr, resolving any delegation designations
+// that may exist.
+func (s *StateDB) ResolveCode(addr common.Address) []byte {
```

Comment:
Can we resolve the code in the EVM?

Reply:
This is a good suggestion, thanks. Done in [01c5d7d](https://github.com/ethereum/go-ethereum/pull/30078/commits/01c5d7d542294106934e14c7d10f80f0d730fbca).